cmd/wego/app/add/cmd.go

```diff
@@ -113,24 +113,28 @@ func runCmd(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	var providerName gitproviders.GitProviderName
+	var normalizedUrl gitproviders.NormalizedRepoURL
 	// We re-use the same --url flag for both git and helm sources.
 	// There isn't really a concept of "provider" in helm charts, and there is nothing to push.
 	// Assume charts are always public and no auth needs to be done.
 	if !isHelmRepository {
-		providerName, err = gitproviders.DetectGitProviderFromUrl(repoUrlString)
```

Comment:
Do we still need the `DetectGitProviderFromUrl` after this? Maybe it can be made private now?

Reply:
we get the providername in NewNormalizedRepoURL(). So instead of doing it twice i just use the normalizedUrl to return it.

Reply:
I guess I am asking if that function needs to be exported?